CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+1. Use the GitHub issue search - Check if the issue has already been reported.
+1. Check if the issue has been fixed - Try to reproduce the issue using the latest [Preview](https://www.visualstudio.com/vs/preview/) or `dev` Build.
+1. Isolate the problem - Create a [minimal, complete, and verifiable example](https://stackoverflow.com/help/mcve) to demonstrate the problem.
+
+A good bug report shouldn’t leave others needing to ask you constantly for more information. Please try to be as detailed as possible in your report. Here are some tips that will help when filing an issue:
+
+* Reproducible steps (1. 2. 3.) and what you expected versus what you actually saw.
+* Images, animations, or a link to a video. **Note: Images and animations illustrate repro-steps but do not replace them.**
+* A code snippet that demonstrates the issue or a link to a code repository we can easily pull down onto our machine to recreate the issue.
+* Any respective build & deployment logs such as Diagnostic Build Output and NuGet Package Manager Console logs.
+* Your environment's version information. You can find this through a command like `Help -> About Visual Studio`
+
+## Suggestions and Feature requests
+
+Suggestions and feature requests are welcomed. Before opening a suggestion or feature request, please take a moment to find out whether your idea fits with the scope and goals of the project. It’s up to you to make a strong case to convince the project maintainers and community the merits of the suggestion or feature. Please provide as much detail and context as possible.
+
+Additionally, do check the existing [Issues](https://github.com/NuGet/Home/issues), [Pull Requests](https://github.com/NuGet/Home/pulls), and [NuGet Designs](https://github.com/NuGet/Home/tree/dev/designs) to see if your idea is already captured. If it is, consider adding feedback to the existing issue, pull request, or contribute directly to the design.
+
+## Pull requests
+Good pull requests of patches, improvements, new features, etc are great contributions. They should however remain focused in scope and avoid unrelated commits.
+
+Before embarking on any significant pull request, please ask a maintainer in a GitHub issue if the scope of the pull request would be reasonable to merge into the project. Otherwise you can potentially waste a lot of time working on something that might not be relevant to the project’s goals.
+
+While creating a pull request, please also include the intent for which you'd like the changes to land. By default, we ask that you pull against the `dev` branch. However if you'd like to see a change in the next service release or preview release of Visual Studio, please communicate with a maintainer of this project to understand if the changes will be able to make the intended release.
+
+## Contributing Fixes
+If you are interested in fixing issues and contributing directly to the code base, please see the document [[Contribute To NuGet|Contribute-to-NuGet]].
```

Comment:
Does this syntax work? The preview didn't turn this into a link

Reply:
Fixed with a link instead! 👍